History.md

```diff
@@ -1,3 +1,49 @@
+# 0.1.6 (04 August 2017)
+* Major Enhancements
+  - Add support for reading HTML tables into DataFrames (@athityakumar)
+  - Add support for importing remote CSVs (@athityakumar)
```

Comment:
I know that this PR is merged, but I'd like to point a minor suggestion. Can attribution please be provided to @anshuman23 too, for working on PR #300 (on which #334 is based on) regarding import of remote CSVs?